src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -289,7 +291,7 @@ internal class SourceMemberFieldSymbolFromDeclarator : SourceMemberFieldSymbol
             DeclarationModifiers modifiers,
             bool modifierErrors,
             DiagnosticBag diagnostics)
-            : base(containingType, modifiers, declarator.Identifier.ValueText, declarator.GetReference(), declarator.Identifier.GetLocation())
+            : base(containingType, modifiers, declarator.Identifier.ValueText ?? "", declarator.GetReference(), declarator.Identifier.GetLocation())
```

Comment:
>declarator.Identifier.ValueText ?? "" [](start = 46, length = 37)

This doesn't feel right.

Reply:
The only `SyntaxToken` that has null `ValueText` is `default(SyntaxToken)`. I think we should change `ValueText` to be a non-nullable property.